.github/workflows/ci.yml

```
@@ -318,33 +347,28 @@ jobs:
 
       - uses: actions/download-artifact@v3
         with:
-          name: LiquidityTests
+          name: Container01
           path: mochawesome-report
       - uses: actions/download-artifact@v3
         with:
-          name: MenuBarTests
+          name: Container02
           path: mochawesome-report
       - uses: actions/download-artifact@v3
         with:
-          name: RewardsPageTests
+          name: Container03
           path: mochawesome-report
       - uses: actions/download-artifact@v3
         with:
-          name: SwapBoxTests
-          path: mochawesome-report
-      - uses: actions/download-artifact@v3
-        with:
-          name: TokenMenuTests
-          path: mochawesome-report
-      - uses: actions/download-artifact@v3
-        with:
-          name: TransactionSettingsTests
+          name: Container04
           path: mochawesome-report
       - uses: actions/download-artifact@v3
+        continue-on-error: true
```

@niemam29 May be use this checks here instead of `continue-on-error: true` so it wont fail if we are not running synpress.
    
```
      - name: Get Commit Message
        run: |
          MSG=$(git log --format=%B -n 1 ${{github.event.after}})
          echo "COMMIT_MSG=$MSG" >> "$GITHUB_ENV"

```
```suggestion
        if: contains(env.COMMIT_MSG, 'trigger-synpress')
```

---

Sadly in that step we're not have checkout to the repo and due to that getting the commit message could be little tricky. This 01 and 02 in names was added to force synpress to execute the tests in order in local.